keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -471,13 +455,22 @@ status:
   - type: Ready
     status: "True"
     lastTransitionTime: "2020-03-30T01:33:51Z"
-  - type: InvalidService
+    reason: "Ready"
+  - type: Valid
     status: "False"
     lastTransitionTime: "2020-03-30T01:33:55Z"
+    reason: "ServiceType"
+    message: "Service type ExternalName is not supported"
   - type: Conflict
     status: "True"
     lastTransitionTime: "2020-03-30T01:33:55Z"
-    message: "Conflicting type. Using \"ClusterSetIP\" from oldest service export in \"cluster-1\". 2/5 clusters disagree."
+    reason: "Conflict"
+    message: "Conflicting type. 2/5 clusters disagree. Using \"ClusterSetIP\" from oldest service export in \"cluster-1\"."
```

Comment:
I think the reason should be "ConflictingType".

Reply:
Yes indeed, although maybe we could extend the reason to do what I wanted to do for LocalConflict instead of encoding the type as well (like having LocalConflict/ExternalConflict) WDYT?

Reply:
So with some inspiration from your message on slack having if we encode having a local conflict in the same condition it would be probably one of these:
```
 - type: Conflict
    status: "True"
    lastTransitionTime: "2020-03-30T01:33:55Z"
    reason: "LocalConflict"
    message: "The local service type conflicts with other constituent clusters. 2/5 clusters disagree. Using \"ClusterSetIP\" from oldest service export in \"cluster-1\"."
 ```

```
 - type: Conflict
    status: "True"
    lastTransitionTime: "2020-03-30T01:33:55Z"
    reason: "ConflictingType"
    message: "The local service type conflicts with other constituent clusters. 2/5 clusters disagree. Using \"ClusterSetIP\" from oldest service export in \"cluster-1\"."
 ```

So if going that route we would have to decide if we want to encode local/external conflict or the "type" of conflict within the reason